moveit_ros/moveit_servo/src/servo_node.cpp

```diff
@@ -50,6 +50,15 @@
 namespace moveit_servo
 {
 
+rclcpp::Time convertClockType(const rclcpp::Time& time, rcl_clock_type_t new_clock_type)
```

Comment:
```suggestion
namespace
{
// This function is used to convert times in case planning_scene_monitor uses RCL_SYSTEM_TIME
rclcpp::Time convertClockType(const rclcpp::Time& time, rcl_clock_type_t new_clock_type)
```